### PR TITLE
Add and remove function

### PR DESCRIPTION
**Pull request by Javivy**
## In this milestone I:

  - [x] Create add function
  - [x] Create remove function
  - [x] Check if the tasks are completed
  - [x] Save the data of the task in the local storage
  - [x] Make use of webpack

### CODE REVIEW

File: src/modules/store.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const changeValue = () => {
+  taskInput.addEventListener('keyup', (e) => {
+    const userTask = taskInput.value.trim();
+    if (e.key === 'Enter' && userTask) {
+      const taskInfo = createObj(userTask, todos.length);
+      todos.push(taskInfo);
+      localStorage.setItem('todo-list', JSON.stringify(todos));
+      displayTask(taskInfo);
+      taskInput.value = '';
+      // eslint-disable-next-line no-restricted-globals
```
Comment (FrederickMih) [inline src/modules/store.js]: - [ ] You are not supposed to disable Linters. These are great tools that flag programming bugs. Please rather fix them.

File: src/modules/displayTask.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+// const todos = JSON.parse(localStorage.getItem('todo-list'));
+
+// function deleteFunction(id) {
+//   console.log(todos);
+//   console.log(id);
+//   // todos = todos.splice(id, 1);
+//   // localStorage.setItem('todo-list', JSON.stringify(todos));
+//   // console.log(todos);
+// }
```
Comment (FrederickMih) [inline src/modules/displayTask.js]: - [ ] Please do not push commented codes to your repo. Comments render your work messy.

File: src/modules/delete.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+const deleteTask = () => {
+  task.forEach((t) => {
+    t.addEventListener('click', (e) => {
+      if (e.target.classList.contains('delete') || e.target.classList.contains('delete-icon')) {
+        todos.splice(t.id, 1);
+        localStorage.setItem('todo-list', JSON.stringify(todos));
+        // eslint-disable-next-line no-restricted-globals
```
Comment (FrederickMih) [inline src/modules/delete.js]: - [ ] Do not disable Linters.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Javivy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :green_circle: 

- PR has a title and summary :+1: 
- Correct GitHub flow :+1: 
- No Linter issue :+1: 
- Great commit history :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Javivy, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**